### PR TITLE
FW-404. Fixed.

### PR DESCRIPTION
**Pull request by changtengfei**

### CODE REVIEW

File: openstack/02a-MAClow/IEEE802154.c [hunk 1]
```diff
@@ -62,7 +62,7 @@ void ieee802154_prependHeader(OpenQueueEntry_t* msg,
    // add termination IE accordingly 
    if (payloadIEPresent == TRUE) {
        ielistpresent = IEEE154_IELIST_YES; 
-       frameVersion  = IEEE154_FRAMEVERSION;
+//       frameVersion  = IEEE154_FRAMEVERSION;
```
Comment (twatteyne) [inline openstack/02a-MAClow/IEEE802154.c]: let's not replace lines by commented lines, but just remove them (applies throughput this PR)

File: openstack/02a-MAClow/IEEE802154.c [hunk 2]
```diff
@@ -168,7 +168,8 @@ void ieee802154_prependHeader(OpenQueueEntry_t* msg,
    temp_8b             |= IEEE154_ADDR_EXT                << IEEE154_FCF_SRC_ADDR_MODE;
    //poipoi xv IE list present
    temp_8b             |= ielistpresent                   << IEEE154_FCF_IELIST_PRESENT;
-   temp_8b             |= frameVersion                    << IEEE154_FCF_FRAME_VERSION;
+   temp_8b             |= IEEE154_FRAMEVERSION            << IEEE154_FCF_FRAME_VERSION;
```
Comment (twatteyne) [inline openstack/02a-MAClow/IEEE802154.c]: please rename this #define to `IEEE154_FRAMEVERSION_2012`

File: openstack/02a-MAClow/IEEE802154.c [hunk 3]
```diff
@@ -168,7 +168,8 @@ void ieee802154_prependHeader(OpenQueueEntry_t* msg,
    temp_8b             |= IEEE154_ADDR_EXT                << IEEE154_FCF_SRC_ADDR_MODE;
    //poipoi xv IE list present
    temp_8b             |= ielistpresent                   << IEEE154_FCF_IELIST_PRESENT;
-   temp_8b             |= frameVersion                    << IEEE154_FCF_FRAME_VERSION;
+   temp_8b             |= IEEE154_FRAMEVERSION            << IEEE154_FCF_FRAME_VERSION;
+   temp_8b             |= IEEE154_SUPPRESSION_YES         << IEEE154_FCF_DSN_SUPPRESSION;
```
Comment (twatteyne) [inline openstack/02a-MAClow/IEEE802154.c]: please rename to `IEEE154_DSN_SUPPRESSION_YES`

**Comment by changtengfei** [comment]
All comments are fixed.

**Comment by twatteyne** [comment]
Thanks a million! I will wait for the Travis builds to pass, then merge.

**Comment by xvilajosana** [comment]
Builds passed. I tested and it is working.

**Comment by twatteyne** [comment]
don't forget to close the issue and delete the branch

**Pull request merged**